src/thumbnailer.cc

```diff
@@ -381,6 +402,7 @@ Thumbnailer::Status Thumbnailer::TryNearLossless(WebPData* const webp_data) {
       return kMemoryError;
     }
     WebPPictureFree(&new_pic);
+    curr_ind++;
```

Comment:
small detail: the style guide requires using ++curr_ind by default (that is, unless you need the semantic of cur_ind++ like you are doing in some other places)
https://google.github.io/styleguide/cppguide.html#Preincrement_and_Predecrement